### PR TITLE
Add WeAct & one of their boards.

### PR DESCRIPTION
**Pull request by bill88t**
Got a WeAct board from their official store, the links are in the new files.
Not in any way affiliated with them.

I am porting CircuitPython to it (PR link: https://github.com/adafruit/circuitpython/pull/8550).

### CODE REVIEW

File: creations/weact.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+## `0x0032_xxxx` - ESP32 dev boards
+
+## `0x0052_xxxx` - S2 dev boards
+
+## `0x0053_xxxx` - S3 dev boards
+
+## `0x00C3_xxxx` - C3 dev boards
+
+## `0x00C6_xxxx` - C6 dev boards
+*  `0x00C6_0001` [WeAct ESP32C6](https://www.aliexpress.com/item/1005005569520224.html)
```
Comment (tannewt) [inline creations/weact.md:16]: Please do one number for each flash variant. (As I said on the CP PR.)

**Review by tannewt — CHANGES_REQUESTED** [review]

File: creations/weact.md (hunk 1 above)
Comment (bill88t) [reply creations/weact.md]: I did, now it's 04 for N4 and 08 for N8.
If there are more C6 boards in the future, we can do 14, 18 etc.

Comment (bill88t) [reply creations/weact.md]: For 16, we could do "a", for 32 "b" and etc.

**Comment by bill88t** [comment]
In case it was missed, I did do the requested change.

**Review by tannewt — APPROVED** [review]
Thank you!

**Pull request merged**